crates/avalanche-types/src/warp/client.rs

```diff
@@ -0,0 +1,43 @@
+use std::io::{Error, ErrorKind, Read, Result};
+use std::str::FromStr;
+
+use crate::ids::Id;
+use crate::proto::pb::warp::{signer_client, SignRequest, SignResponse};
+use prost::bytes::Bytes;
+use tonic::transport::Channel;
+
+#[derive(Clone)]
+pub struct WarpSignerClient {
+    inner: signer_client::SignerClient<Channel>,
+}
+
+impl WarpSignerClient {
+    pub fn new(client_conn: Channel) -> Self {
+        Self {
+            inner: signer_client::SignerClient::new(client_conn)
+                .max_decoding_message_size(usize::MAX)
+                .max_encoding_message_size(usize::MAX),
```

Comment:
These are already the defaults

```suggestion
            inner: signer_client::SignerClient::new(client_conn),
```